doc-content/enterprise-only/cases/case-management-clean-case-log-proc.adoc

```diff
@@ -0,0 +1,61 @@
+[id='case-management-clean-case-log-proc']
+= Removing case logs from the database
+
+Use the `CaseLogCleanupCommand` to cleanup cases, such as canceled cases that lack value and that are using database space. The `CaseLogCleanupCommand` command consists of logic to clean up all or selected cases automatically. By default, the `CaseLogCleanupCommand` is executed once a day but can be configured to run on different intervals.
```

Comment:
What does "lack value" mean? Literally has no values in it or that it is not relevant anymore?
"using up database space"?
The command consists of logic or contains logic?

Reply:
Rewritten as: 

Use the `CaseLogCleanupCommand` to clean-up cases, such as canceled cases that are using up database space. The `CaseLogCleanupCommand` command contains logic to automatically clean-up all or selected cases.